scan_topic_name: /scan
scan_queue_size: 1
